amazon-chime-sdk/src/main/java/com/amazonaws/services/chime/sdk/meetings/audiovideo/audio/AudioMode.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazonaws.services.chime.sdk.meetings.audiovideo.audio
+
+/**
+ * [AudioMode] describes the audio mode in which the audio client should operate during a meeting session.
+ */
+enum class AudioMode(val value: Int) {
```

Comment:
I am wondering why we do not support Stereo16K. It should work.

Reply:
There was a discussion a while ago. The main concern is that stereo at 16kHz sampling rate does not have good quality, and could cause confusion to builders.